terraform/modules/azure-jenkinsinfra-azurevm-agents/main.tf

```
@@ -78,7 +78,11 @@ resource "azurerm_network_security_rule" "allow_outbound_ssh_from_ephemeral_agen
   source_port_range           = "*"
   source_address_prefixes     = data.azurerm_subnet.ephemeral_agents.address_prefixes
   destination_port_range      = "22"
-  destination_address_prefixes = split(" ", local.github_destination_address_prefixes)
+  #Filter only for ipv4 ips
+  destination_address_prefixes = [
+    for ip in split(" ", local.github_destination_address_prefixes) : ip
+    if can(cidrnetmask("${ip}/32"))
```

```suggestion
    if can(cidrnetmask(ip))
```

=> otherwise the string has twice the suffix `/32`